bot/exts/filters/filtering.py

```diff
@@ -438,7 +442,7 @@ async def _send_log(
         content = str(msg.author.id)  # quality-of-life improvement for mobile moderators
 
         # If we are going to autoban, we don't want to ping and don't need the user ID
-        if reason and "[autoban]" in reason:
+        if auto_ban:
```

Comment:
Did the previous approach not work for invite filters since it could contain multiple reasons? In that case, wouldn't the type annotation of the `reason` argument for this function be wrong, or are the reasons concatenated before this function is called?

Reply:
for guild_invites, reason will always be passed as None.

It's initialised as None here https://github.com/python-discord/bot/blob/main/bot/exts/filters/filtering.py#L338 and then since _has_invites doesn't return a tuple, this conditional means it is never updated.

https://github.com/python-discord/bot/blob/main/bot/exts/filters/filtering.py#L340-L343